assets/src/edit-story/components/tablist/styles.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_VARIANTS,
+  Headline,
+  ThemeGlobals,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../design-system';
+
+export const Tablist = styled.div`
+  width: 308px;
+  background: ${({ theme }) => theme.colors.bg.primary};
+`;
+Tablist.propTypes = {
+  // label to describe purpose of tabs
+  'aria-label': PropTypes.string.isRequired,
+};
+
+export const PanelWrapper = styled.div`
+  width: 100%;
+  background: ${({ theme }) => theme.colors.bg.primary};
+
+  ${({ isExpanded }) =>
```

Comment:
i think this needs 
```suggestion
  ${({ isExpanded, theme }) =>
```

and then on line 57 `${theme.colors.bg.primary}`